packages/field-base/src/input-mixin.d.ts

```diff
@@ -50,4 +50,8 @@ export declare class InputMixinClass {
   protected _toggleHasValue(value: boolean): void;
 
   protected _valueChanged(value?: string, oldValue?: string): void;
+
+  private __inputValuePopulatedChanged(): void;
+
+  private __onChange(event: void): void;
```

Comment:
We don't need private methods to be listed in `.d.ts` as they are not supposed to be overridable.

Reply:
Removed.